tests/scripts/thread-cert/Cert_5_5_05_SplitMergeREED.py

```diff
@@ -101,7 +101,7 @@ def test(self):
         self.nodes[ROUTER3].stop()
         time.sleep(140)
 
-        self.assertEqual(self.nodes[ROUTER1].get_state(), 'child')
+        self.assertEqual(self.nodes[ROUTER1].get_state(), 'router')
```

Comment:
it might be reasonable for old routers to try to keep old status firstly when merging? @jwhui